src/main/java/ch/njol/skript/expressions/ExprLocationFromVector.java

```diff
@@ -46,11 +48,10 @@
 public class ExprLocationFromVector extends SimpleExpression<Location> {
 
 	static {
-		// TODO fix slowdowns and enable again, for now nuked for greater good
-//		Skript.registerExpression(ExprLocationFromVector.class, Location.class, ExpressionType.SIMPLE,
-//				"%vector% [to location] [in] %world%", "location (from|of) %vector% [(from|in)] %world%",
-//				"%vector% [to location] [in] %world% with yaw %number% and pitch %number%",
-//				"location (from|of) %vector% [(in|from)] %world% with yaw %number% and pitch %number%");
+		Skript.registerExpression(ExprLocationFromVector.class, Location.class, ExpressionType.SIMPLE,
+				"%vector% [to location] in %world%", "location (from|of) %vector% in %world%",
+				"%vector% [to location] in %world% with yaw %number% and pitch %number%",
+				"location (from|of) %vector% in %world% with yaw %number% and pitch %number%");
```

Comment:
```suggestion
		Skript.registerExpression(ExprLocationFromVector.class, Location.class, ExpressionType.SIMPLE,
				"%vector% [to location] in %world%",
				"location (from|of) %vector% in %world%",
				"%vector% [to location] in %world% with yaw %number% and pitch %number%",
				"location (from|of) %vector% in %world% with yaw %number% and pitch %number%"
		);
```

Reply:
re: converters
I don't think a converter can take in both a world and a vector, it could probably us a default world but I don't believe you'd ever be able to specify the world.

Reply:
ah yes that was a silly oversight on my end... ignore what i said!